index.html

```diff
@@ -173,7 +178,33 @@ <h4 class="card-header">Card Title</h4>
         </div>
       </div>
       <!-- /.row -->
-  
+
+
+      <!-- Add a favorite button-->
+      <form>   
+        <div id="scrollbox" class="" sytle="display: block">
+            <div id="1">Option 1 <input type='checkbox' class='handleFavorite'>Favorite</div>
+            <div id="2">Option 2 <input type='checkbox' class='handleFavorite'>Favorite</div>
+            <div id="3">Option 3 <input type='checkbox' class='handleFavorite'>Favorite</div>
+            <div id="4">Option 4 <input type='checkbox' class='handleFavorite'>Favorite</div>
+            <div id="5">Option 5 <input type='checkbox' class='handleFavorite'>Favorite</div>
+        </div>  
+      </form>
+
+       <!-- Search user ratings-->
+       <div class="trail-list">
+        <ul>
+                <li>
+                    <img src=""alt="appIcon" height="60" width="60">
+                    <a href="">Hiking trails</a>
+                    <figcaption><i>Rate your experience on the trail</i></figcaption>
+                  <div class="rating">
+                    <span style=" color: #FDE16D;"  class="fa fa-star"></span>
+                    <span style=" color: #FDE16D;" class="fa fa-star "></span>
+                    <span style=" color: #FDE16D;" class="fa fa-star "></span>
+                    <span style=" color: #FDE16D;" class="fa fa-star"></span>
+                    <span style=" color: #FDE16D;" class="fa fa-star"></span>
+                </div>
```

Comment:
maybe add another span to say this is a total out of 5 either going left or right. [https://gyazo.com/3389dc6374ea40f5fd20f0cd684ec617](url)

Reply:
Sounds good, I will-- thanks you.